pkg/common/webhook.go

```diff
@@ -1,10 +1,22 @@
 package common
 
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
 const (
 	WebhookName        = "fluid-pod-admission-webhook"
 	WebhookServiceName = "fluid-pod-admission-webhook"
-	WebhookServicePath = "/mutate-v1-pod"
+	WebhookServicePath = "/mutate-fluid-io-v1alpha1-schedulepod"
```

Comment:
unused constant, it has been moved to pkg/webhook/scheduler/mutating/webhook.go

Reply:
Done